pkg/maps/cells.go

```diff
@@ -21,6 +23,12 @@ var Cell = cell.Module(
 	// ConfigMap stores runtime configuration state for the Cilium datapath.
 	configmap.Cell,
 
+	// Receives datapath signals for GC fill-up events
+	gc.Cell,
```

Comment:
You mean add an intermediary `ctmap/cell.go` so as not refer to the `gc.Cell` directly here? Or something else?

Reply:
Unfortunately this would create an import cycle, as `ctmap/gc` by definition needs to import `ctmap`. This means that a `ctmap/cell.go` can not import `ctmap/gc`.

I added a note in `maps/cell.go` to this effect.

Reply:
might be worth to ad the `gc.go` into the same package.

Reply:
fine for me :+1: